test/integration/serverless/serverless.yml

```diff
@@ -1,5 +1,5 @@
 # IAM permissions require service name to begin with 'integration-tests'
-# Adding a comment to start integration tests on push
+# Adding a comment to start integration tests
```

Comment:
Can you make another commit removing this comment? Also, the integration tests _should_ start automatically when you make a change to one of the other test files...